src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -128,10 +128,17 @@ internal static ProcessInfo CreateProcessInfo(Interop.procfs.ParsedStat procFsSt
         {
             int pid = procFsStat.pid;
 
+            // Get long process name if possible, otherwise use a fall back method.
+            string procName = Path.GetFileName(Process.GetExePath(pid));
+            if (String.IsNullOrEmpty(procName))
```

Comment:
Nit: String => string

Reply:
Yes correct however functionality is the same here.

Reply:
interesting post on it https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html

Reply:
Yes normally I would use `string` I must of mistyped. Basic types vs complex, `string` is fine but `DateTime` is a complex type etc.